client/jetpack-onboarding/style.scss

```diff
@@ -13,11 +13,63 @@
 		}
 	}
 
+	.steps__summary-columns {
+			display: flex;
+			flex-flow: row wrap;
+			align-items: flex-start;
+
+			margin-top: 30px;
+
+			@media ( max-width: 700px ) {
+				margin-top: 0;
+			}
+		}
+
+	.steps__summary-column {
+		flex: 1;
+		justify-content: center;
+		flex-direction: column;
+		text-align: center;
+		margin-bottom: 40px;
+		flex-wrap: wrap;
+		font-weight: bold;
+		margin-top: 20px;
+		margin-bottom: 20px;
+
+		@media ( max-width: 700px ) {
```

Comment:
We should not be introducing custom breakpoints, see `_breakpoints.scss` for existing ones.

Reply:
true 👍